app/controllers/admin/categories_controller.rb

```diff
@@ -17,5 +28,10 @@ def destroy
       category.destroy
       redirect_to admin_categories_path
     end
+
+    private
```

Comment:
I'm not sure what this is referring too... there is a blank line berfore and after!!!!